VIC_RBM/model_run/result_analysis/scripts/plot_flow_future.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/local/anaconda/bin/python
+
+# This script plots different routed flow results together
+
+import numpy as np
+import datetime as dt
+import matplotlib.pyplot as plt
+import sys
+import pandas as pd
+import argparse
+import my_functions
+
+parser = argparse.ArgumentParser()
```

Comment:
Maybe instead of reading arguments via command line arguments, you could try using a configuration file. I started using the read_config function that Joe wrote in some of my scripts. The function (see 
https://github.com/orianac/BPA_304/blob/feature/sensitivity/tools/sensitivity/run_sensitivity.py) uses the SafeConfigParser and OrderedDict modules. It reads templates like this one https://github.com/orianac/BPA_304/blob/feature/sensitivity/tools/sensitivity/run_prms_sensitivity.cfg.template